### ISSUE TITLE
TypeScript definition errors in v0.68.1

### ISSUE THREAD

**Issue by mihai-dinculescu** [Semantic-Org/Semantic-UI-React#1620]
Greetings,

After upgrading to the latest version I get the following error:

```
ERROR in node_modules\semantic-ui-react\dist\commonjs\addons\Confirm\Confirm.d.ts
(4,18): error TS2430: Interface 'ConfirmProps' incorrectly extends interface 'ModalProps'.
  Property 'content' is optional in type 'ConfirmProps' but required in type 'ModalProps'.

```

**Comment by layershifter** [comment]
There's a PR open for this, #1611. Duplicate #1612. Thanks for the report.

**Comment by levithomason** [comment]
Whew, I should get that shipped right away ;)

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.68.2`.
